crates/pathfinder/src/rpc/v02/method/block_hash_and_number.rs

```diff
@@ -0,0 +1,62 @@
+use anyhow::Context;
+use stark_hash::StarkHash;
+
+use crate::{rpc::v02::RpcContext, storage::StarknetBlocksTable};
+
+type BlockNumber = u64;
+
+#[derive(serde::Serialize)]
+pub struct BlockNumberResult {
+    pub block_hash: StarkHash,
+    pub block_number: BlockNumber,
```

Comment:
I would use the types from `core` for this, which are new-types around these primitives (and are just more descriptive).
```suggestion
    pub block_hash: StarknetBlockHash,
    pub block_number: StarknetBlockNumber,
```

Reply:
Done